src/UIBaseModules.ts

```diff
@@ -11,3 +11,7 @@ export { IResultsComponentBindings } from './ui/Base/ResultsComponentBindings';
 export { Initialization } from './ui/Base/Initialization';
 export { Initialization as CoveoJQuery } from './ui/Base/Initialization';
 export { initCoveoJQuery } from './ui/Base/CoveoJQuery';
+export { ResponsiveComponentsManager } from './ui/ResponsiveComponents/ResponsiveComponentsManager';
```

Comment:
As explained in Slack, there should be accompanying tests when something is exposed publically like this, because it means that external code depends on it.

The tests should aim to test and ensure the backward compatibility/public function signature so that we don't "break" the salesforce integration 6 months from now.